src/app/cmdoptions.cpp

```diff
@@ -427,6 +429,13 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
                     throw CommandLineParameterError(QObject::tr("%1 must specify a valid port (1 to 65535).")
                                                     .arg(u"--webui-port"_qs));
             }
+            else if (arg == TORRENTING_PORT_OPTION)
+            {
+                result.torrentingPort = TORRENTING_PORT_OPTION.value(arg);
+                if ((result.torrentingPort < 1) || (result.torrentingPort > 65535))
+                    throw CommandLineParameterError(QObject::tr("%1 must specify a valid port (1 to 65535).")
+                                                    .arg(u"--torrenting-port"_qs));
```

Comment:
```suggestion
                if ((result.torrentingPort < 1) || (result.torrentingPort > 65535))
                {
                    throw CommandLineParameterError(QObject::tr("%1 must specify a valid port (1 to 65535).")
                                                    .arg(u"--torrenting-port"_qs));
                }
                result.torrentingPort = TORRENTING_PORT_OPTION.value(arg);
```

Reply:
Dear Chocobo1! Is this correct? I made the changes, but if I assign the variable after the `if()` call, the `if()` will always fail, because it's not yet assigned. Just a question though. Don't underestimate my power-level when it comes to being awful at this. Thank you!

Reply:
Oops my bad, forget the reordering... just add the brackets:
```suggestion
                result.torrentingPort = TORRENTING_PORT_OPTION.value(arg);
                if ((result.torrentingPort < 1) || (result.torrentingPort > 65535))
                {
                    throw CommandLineParameterError(QObject::tr("%1 must specify a valid port (1 to 65535).")
                                                    .arg(u"--torrenting-port"_qs));
                }
```